test/testdrive/status-history.td

```diff
@@ -63,6 +66,12 @@ $ kafka-verify-data format=avro sink=materialize.public.kafka_sink sort-messages
 "<TIMESTAMP> UTC" ${sink_id} starting <null> <null>
 "<TIMESTAMP> UTC" ${sink_id} running <null> <null>
 
+> select * from mz_internal.mz_sink_status where id = '${sink_id}';
+"${sink_id}" kafka_sink kafka "<TIMESTAMP> UTC" running <null> <null>
+
 > select * from mz_internal.mz_source_status_history where source_id = '${source_id}' order by occurred_at;
 "<TIMESTAMP> UTC" ${source_id} starting <null> <null>
 "<TIMESTAMP> UTC" ${source_id} running <null> <null>
+
+> select * from mz_internal.mz_source_status where id = '${source_id}';
```

Comment:
What do you think about adding tests that we do in fact get results from multiple sources/sinks?

Reply:
In the sense of: create a second source and sink, and confirm that we see those in the status tables too?

Reply:
That's right!

Reply:
Done!